.github/actions/check-branch-commits/README

```diff
@@ -0,0 +1,13 @@
+## Description
+
+Runs the branch commit checker on aptos-core, assuming it is already checked out in the current working directory.
```

Comment:
I think this sentence by itself doesn't really tell me much. I'd recommend you more-or-less copy and paste the description from the PR into this README.
Also please add a `.md` otherwise this won't be properly recognized as markdown.

Reply:
Actually frankly I don't think you need to repeat the inputs and that this is a composite action again in the readme since this info is already available in the action.yaml and it's kinda annoying to keep them manually in sync.
I'd recommend to only put into README info that is not available in the yaml, such as a longer description of what this action does. Alternatively you can also include the longer decription of what this action does in the yaml and delete the README altogether.

Reply:
Yeah, agree that it's probably better to just delete it here. This was simply just copy/pasta from the new GHA migration that's happening, e.g., here: https://github.com/aptos-labs/actions. But I've removed it in this PR for now (and from `fullnode-sync`).